db.py

```diff
@@ -34,6 +39,7 @@ class Outcome(Base):
     game_id = db.Column(db.Integer, db.ForeignKey("games.id"))
     result = db.Column(db.Integer)
     runs = db.Column(db.Integer)
+    game = relationship("Game", back_populates="outcomes")
```

Comment:
I think you don't need to declare this, as `back_populates` on line 27 takes care of generating the other side of the relationship?

Reply:
That's what I thought, but querying games from my terminal threw an exception that suggested otherwise. [See pastebin for (long) traceback](https://pastebin.com/eAf1x8MC)

Reply:
Hmm. I wonder if it has something to do with the order that the classes are defined in? /shrug something to figure out later.

Reply:
Looks like the answer was `backref` instead of `back_populates` - https://docs.sqlalchemy.org/en/14/orm/backref.html#relationships-backref